slam/CMakeLists.txt

```diff
@@ -1,198 +1,45 @@
 cmake_minimum_required(VERSION 3.0.2)
 project(slam)
 
-## Compile as C++11, supported in ROS Kinetic and newer
-# add_compile_options(-std=c++11)
+add_compile_options(-std=c++14)
 
 ## Find catkin macros and libraries
 ## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
 ## is used, also find other catkin packages
+
+
+
+ #find_package(Eigen3 REQUIRED)
+
+list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/builds/cmake)
+
+
 find_package(catkin REQUIRED COMPONENTS
   roscpp
   rospy
   std_msgs
-  nav_msgs
-  geometry_msgs
-  message_generation
```

Comment:
this all should be fine, just make sure you've done a clean + build before merging since getting rid of some of this (esp message_generation) seems like it could cause issues with message files not being built.

Reply:
Yep. We did a successful build.